go/tools/builders/stdlib.go

```diff
@@ -154,11 +154,23 @@ You may need to use the flags --cpu=x64_windows --compiler=mingw-gcc.`)
 			break
 		}
 	}
+	absoluteLdFlags, nonAbsoluteLdFlags, err := splitAbsEnvVar(os.Getenv("CGO_LDFLAGS"), cgoAbsEnvFlags)
+	if err != nil {
+		return fmt.Errorf("error splitting cgo ldflags: %v", err)
+	}
+	os.Setenv("CGO_LDFLAGS", nonAbsoluteLdFlags)
+	if len(absoluteLdFlags) > 0 {
+		ldflags = append(ldflags, "-extldflags")
+		splitFlags := strings.Split(absoluteLdFlags, " ")
```

Comment:
We are splitting the flags we just joined. Instead, could we have the helper function return string slices?